test/letters/components/UpdateFailureAlert.unit.spec.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactTestUtils from 'react-dom/test-utils';
+import { expect } from 'chai';
+
+import { getFormDOM } from '../../util/schemaform-utils';
+
+import UpdateFailureAlert from '../../../src/js/letters/components/UpdateFailureAlert';
+
+const defaultProps = {
+  addressObject: {
+    addressOne: '123 Main St N',
+    city: 'Bigtowne',
+    stateCode: 'BS',
+    countryName: 'Elsweyre'
+  }
+};
+
+describe('<UpdateFailureAlert>', () => {
+  it('should render address and warning', () => {
+    // Have to wrap the stateless (function) component in <div></div> so renderIntoDocument() is happy
```

Comment:
Good catch! I'm surprised that none of the other teams have run into this problem before. I guess you could also abstract this away by making a wrapping class-based component (e.g., `functionalRender()` that just renders its children) somewhere in the common test helpers but that's probably overkill if the alternative is to wrap things in a `<div/>`.

Reply:
Heh, I tried doing something like that first, but when I found I could just wrap the silly thing in a `<div>` I nixed it for a few reasons:

1) Because I don't have to write the thing
2) When I write the tests, it's still easier to wrap it in a `<div>` than to import and use that component
3) It's less likely to break in weird, unexpected ways because of the testing wrapper